src/examples/example_print_metadata.rs

```diff
@@ -38,10 +37,11 @@ fn main() {
     let meta = Metadata::try_from(api.get_metadata().unwrap()).unwrap();
 
     meta.print_overview();
-    meta.print_modules_with_calls();
-    meta.print_modules_with_events();
+    meta.print_pallets();
+    meta.print_pallets_with_calls();
+    meta.print_pallet_with_events();
     meta.print_modules_with_errors();
```

Comment:
Here you can also change modules to pallets. :)

Reply:
Ouch ;)